README.md

```diff
@@ -1,38 +1,137 @@
-# react-apollo
+# [React Apollo](http://dev.apollodata.com/react/) [![npm version](https://badge.fury.io/js/react-apollo.svg)](https://badge.fury.io/js/react-apollo) [![Get on Slack](https://img.shields.io/badge/slack-join-orange.svg)](http://www.apollostack.com/#slack)
 
-[![npm version](https://badge.fury.io/js/react-apollo.svg)](https://badge.fury.io/js/react-apollo)
-[![Get on Slack](https://img.shields.io/badge/slack-join-orange.svg)](http://www.apollostack.com/#slack)
-[![Build Status](https://travis-ci.org/apollographql/react-apollo.svg?branch=master)](https://travis-ci.org/apollographql/react-apollo)
-[![Coverage Status](https://coveralls.io/repos/github/apollographql/react-apollo/badge.svg?branch=master)](https://coveralls.io/github/apollographql/react-apollo?branch=master)
```

Comment:
If you're concerned about build status sometimes going red, how about we just put coverage there? I think that rarely accidentally goes to zero. I actually think it's also quite rare for the build on master to fail, so I'm not too worried about having both badges there.